libs/redis/langchain_redis/version.py

```diff
@@ -0,0 +1,3 @@
+__version__ = "0.0.2"
+__lib_name__ = f"langchain-redis_v{__version__}"
+__with_redis_py_only_lib_name__ = f"redis-py({__lib_name__})"
```

Comment:
the only class I think we'd need this for is the chat message history, but that's only temporary as we should plan to replace the guts of that with the StandardSessionManager class from RedisVL too